XCUITests/HomePageSettingsUITest.swift

```diff
@@ -122,6 +122,19 @@ class HomePageSettingsUITests: BaseTestCase {
 
         XCTAssertEqual("Enter a webpage", value as! String)
     }*/
+    
+    func testSetFirefoxHomeAsHome() {
+        navigator.goto(HomeSettings)
+        navigator.performAction(Action.SelectHomeAsHistoryPage)
```

Comment:
Not sure if it would be a good idea open a new tab here and check that the history page is shown, then change and be sure that it is updated with the new setting for home...